react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+type topics = {
+  topic: string;
+  id: number;
+  information: string;
+};
+
+type Props = {
+  topicArray: topics[];
+};
+
+export default function Accordion({ topicArray }: Props) {
+  return (
+    <div>
+      <Topics topicArray={topicArray} />
+    </div>
+  );
+}
+
+function Topics({ topicArray }: Props) {
+  const [clickedTopic, setClickedTopic] = useState('');
+
+  const topicMarkup = topicArray.map((item) => (
+    <li
```

Comment:
The constraint listed in the instructions was intended to have you take all this code (lines 26-44) and put it in a separate component, say, TopicCard.